elastic/integrations
# added metrics data streams to fleet_server

## Proposed commit message



## Checklist


## Author's Checklist



## How to test this PR locally



Start kibana locally and upload the new version of the package from `/build/packages` folder (or use elastic-package)
```
curl -XPOST -H 'content-type: application/zip' -H 'kbn-xsrf: true' http://localhost:5601/api/fleet/epm/packages -u elastic:changeme --data-binary @fleet_server-1.4.0.zip
```

The package installs successfully and the index templates are created with the right fields and ilm policies.

<img width="1873" alt="image" src="https://github.com/elastic/integrations/assets/90178898/132adc23-bc5e-47b6-8e6c-9a63bdfdbb85">

The data streams are going to be populated by a new kibana task, see [this pr](https://github.com/elastic/kibana/pull/168435)


## Related issues


- 

## Screenshots
